examples/pendulum/pendulum.bioMod

```diff
@@ -0,0 +1,30 @@
+version 3
+gravity 0 0 -9.81
+
+
+// DEFINITION DU SEGMENTS
+
+// Seg1
+    segment Seg1
+        translations	y
+        rotations	x
+        ranges -5 5
+                -pi pi
+
+	mass 1
+	inertia 
+	  1 0 0 
+	  0 1 0
+	  0 0 1
+	com 0 0 -1
+	mesh 0 0 0
+	mesh  0 0 -1
+endsegment
+	
+// Marker on Seg1
+    marker m0
+        parent Seg1
+        position 0 0 0
+    endmarker
```

Comment:
Is there a point for this marker? It feels that it would make more sense to be at 0 0 -1, so it shows the end of the segment